Makefile

```diff
@@ -1,29 +1,90 @@
 APP_NAME=ocp4-openscap-content
+NAMESPACE=openshift-compliance
 
 REPO?=quay.io/jhrozek
 CNT_RUNTIME?=podman
 
 CONTENT_REPO?=https://github.com/ComplianceAsCode/content
-CONTENT_BRANCH?=master
-
-.PHONY: build build-nocache tag-latest tag-branch push-latest push-branch
+CONTENT_BRANCH?=$(shell git rev-parse --abbrev-ref HEAD)
+TAG?=latest
 
 all: build
 
+.PHONY: build
 build:
 	$(CNT_RUNTIME) build --build-arg=repo=${CONTENT_REPO} --build-arg=branch=${CONTENT_BRANCH} -f Dockerfile -t $(APP_NAME)
 
+.PHONY: build-nocache
 build-nocache:
 	$(CNT_RUNTIME) build --build-arg=repo=${CONTENT_REPO} --build-arg=branch=${CONTENT_BRANCH} --no-cache -f Dockerfile -t $(APP_NAME)
 
+.PHONY: build-dev
+build-dev: check-content-path
+	$(eval CONTENT_NAME = $(notdir ${CONTENT_PATH}))
+	cp $(CONTENT_PATH) $(CONTENT_NAME)
+	-$(CNT_RUNTIME) build --build-arg=content=${CONTENT_NAME} -f Dockerfile.dev -t $(APP_NAME)
```

Comment:
why not add a tag already here?

Reply:
Good call. I added `:$(TAG)` to all the `build-*` targets and amended the Makefile a little.